src/utils/packageUtils.ts

```diff
@@ -47,6 +47,9 @@ export type IdRegistryValue = { prefix: string; label: string };
 export type IdRegistry = {
   [key: string]: IdRegistryValue;
 };
+export type UnzipResults = {
```

Comment:
This smelled funny but I read that having an async function return a boolean promise was bad (don't remember the reason unfortunately – please set me straight if this is dumb) and at one point I was thinking about including a manifest of unzipped files in the return value – which I clearly didn't end up doing so the value of this object is more iffy.

Reply:
Yeah, I would either return a manifest of files or some jsZip thing.  I'd have to look at the SDR code that handles retrieves but that general pattern should be followed to keep it all in memory until the source format files are written.

Reply:
Removed entirely since we're not processing the zip in memory.